redbot/cogs/trivia/trivia.py

```diff
@@ -512,6 +568,66 @@ def get_trivia_list(self, category: str) -> dict:
             else:
                 return dict_
 
+    async def _save_trivia_list(self, ctx: commands.Context, file: discord.Attachment):
+        """Checks and saves a trivia list to data folder.
+
+        Parameters
+        ----------
+        file : discord.Attachment
+            A discord message attachment.
+
+        Returns
+        -------
+        None
+        """
+        filename = file.filename.rsplit(".", 1)[0]
+
+        buffer = io.BytesIO(await file.read())
+        yaml.safe_load(buffer)
+        buffer.seek(0)
+
+        file = cog_data_path(self) / f"{filename}.yaml"
+
+        basefileexists = None
```

Comment:
Nitpick, I think using only bool type for this variable makes more sense here
```suggestion
        basefileexists = False
```